create-system-users.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Figuring out where we are
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
```

Comment:
```suggestion
DIR=$(dirname $(realpath ${BASH_SOURCE[0]}))
```

(Is this not always going to be `/local` on our systems? Do we even need to calculate it?)